gcf_data_mapper/read.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import json
+from typing import Any, Optional, Union
+
+import click
+
+
+def read_csv(file_path: str) -> list[dict[str, Any]]:
+    """
+    Reads a csv file and returns a list of dictionaries
+
+    :param str: a file path to the csv file
+    :return list: A list of strings
+    """
+    with open(file_path, "r") as file:
+        csv_reader = csv.DictReader(file)
+        data = [
+            {
+                "country": line["country"],
+                "capital": line["capital"],
+                "avg_temp_celsius": float(line["avg_temp_celsius"]),
+                "annual_rainfall_mm": int(line["annual_rainfall_mm"]),
+                "climate_zone": line["climate_zone"],
+            }
+            for line in csv_reader
+        ]
+        return data
+
+
+def read_json(file_path: str) -> dict:
+    """
+    Reads a json file, and returns a dict
+
+    :param str: A file path to the csv file
+    :return dict: A dictionary of the json data
+    """
+    with open(file_path, "r") as file:
+        return json.load(file)
+
+
+def read_data_file(
+    file_path: str,
+) -> Optional[Union[dict[str, Any], list[dict[str, Any]]]]:
+    """
+    Simple program that reads a data file,
+    calls a function to read a csv or json file respectively
+
+    :param str: A file path to the csv/json file
```

Comment:
Missing the parameter name, please see [here](https://github.com/climatepolicyradar/navigator-admin-backend/blob/2674536196424175743efe20d72af0cc8cd2942f/app/repository/document.py#L201) for example